onisite/settings_base.py

```diff
@@ -184,6 +184,7 @@
 # Suggested order: https://docs.djangoproject.com/en/1.9/ref/middleware/#middleware-ordering
 MIDDLEWARE_CLASSES = (
     'django.middleware.security.SecurityMiddleware',
+    'core.middleware.DisableClientSideCachingMiddleware',              # OpenONI
```

Comment:
your commit mentions that this should be disabled for production....you mean that the DisableClientSideCachingMiddleware should be disabled / commented out?  Or that the client side caching should be disabled and therefore left in the settings_base file?

Reply:
I mean that this middleware should be disabled in production. I'm going to work on splitting Django settings for dev and production as another PR to follow this one.

Reply:
Addressed in #361